src/utils/url-test.js

```diff
@@ -348,3 +350,33 @@ describe('getCanonicalUrl', () => {
     expect(pageQuery).to.equal("link[rel='canonical']");
   });
 });
+
+describe('isSecure', () => {
+  it('should default to current page', () => {
```

Comment:
Could you expand on these test descriptions, just to avoid potential confusion?

For instance, "should default to current page" could be expanded to "first parameter should default to current page"

And "should output true" could be expanded to "should output true when passed a secure URL"

Reply:
Did you have a chance to respond to this comment? I noticed it was resolved a couple times without a comment or code changes

Reply:
I'm not sure why but the file didn't update in git until I added more spacing.  It had been updated locally already.  Very odd.